src/pages/help-centre/deriv-bot.tsx

```diff
@@ -4,9 +4,14 @@ import dbot from './data/_dbot'
 import { WithIntl } from 'components/localization'
 import { SEO } from 'components/containers'
 import { TGatsbyHead } from 'features/types'
+import useRegion from 'components/hooks/use-region'
+import PageNotFound from 'features/pages/404'
 
-const DBot = () => <QuestionsTemplate data={dbot} />
+const DBot = () => {
+    const is_eu = useRegion()
 
+    return !is_eu ? <QuestionsTemplate data={dbot} /> : <PageNotFound />
```

Comment:
we have a component call protected route to handle this can you please try to change using that component